packages/components/src/styles/theme.js

```diff
@@ -2,35 +2,39 @@ const colors = require('./colors');
 
 module.exports = {
   fontFamily: {
-    sans: ['Roboto'],
-    'headline-1': 'Roboto',
-    'headline-2': 'Roboto',
-    'headline-3': 'Roboto',
-    'headline-4': 'Roboto',
-    'headline-5': 'Roboto',
-    'headline-6': 'Roboto',
-    body: 'Roboto',
-    detail: 'Roboto',
-    eyebrow: 'Roboto',
-    form: 'Roboto',
-    formError: 'Roboto',
+    sans: ['Roboto-Regular'],
+    'headline-1': 'Roboto-Bold',
+    'headline-2': 'Roboto-Light',
+    'headline-3': 'Roboto-Regular',
+    'headline-4': 'Roboto-Regular',
+    'headline-5': 'Roboto-Bold',
+    'headline-6': 'Roboto-Medium',
+    body: 'Roboto-Regular',
+    detail: 'Roboto-Regular',
+    eyebrow: 'Roboto-Regular',
+    form: 'Roboto-Regular',
+    formError: 'Roboto-Regular',
     formLabel: 'Roboto-Bold',
-    button: 'Roboto',
-    link: 'Roboto',
-    'm-headline-1': 'Roboto',
-    'm-headline-2': 'Roboto',
-    'm-headline-3': 'Roboto',
-    'm-headline-4': 'Roboto',
-    'm-headline-5': 'Roboto',
-    'm-headline-6': 'Roboto',
-    'm-body': 'Roboto',
-    'm-detail': 'Roboto',
-    'm-eyebrow': 'Roboto',
-    'm-form': 'Roboto',
-    'm-formError': 'Roboto',
+    button: 'Roboto-Medium',
+    link: 'Roboto-Regular',
+    'm-headline-1': 'Roboto-Bold',
+    'm-headline-2': 'Roboto-Light',
+    'm-headline-3': 'Roboto-Regular',
+    'm-headline-4': 'Roboto-Regular',
+    'm-headline-5': 'Roboto-Bold',
+    'm-headline-6': 'Roboto-Medium',
+    'm-body': 'Roboto-Regular',
+    'm-detail': 'Roboto-Regular',
+    'm-eyebrow': 'Roboto-Regular',
+    'm-form': 'Roboto-Regular',
+    'm-formError': 'Roboto-Regular',
     'm-formLabel': 'Roboto-Bold',
-    'm-button': 'Roboto',
-    'm-link': 'Roboto',
+    'm-button': 'Roboto-Medium',
+    'm-link': 'Roboto-Regular',
+    'intensity-light': 'Roboto-Light',
+    'intensity-regular': 'Roboto-Regular',
+    'intensity-medium': 'Roboto-Medium',
+    'intensity-bold': 'Roboto-Bold',
```

Comment:
just double checking, are those the values expected by design?

Reply:
I believe they don't have a name for that.

Reply:
I mean, are those the styles expected by design?

Reply:
These values are defined on DSM with no clear naming convention. 

H6 -> tag
Dark -> TypographyMode
100 -> TypographyMode

Type face `Roboto Medium` is our new attribute `intensity`

![Screenshot 2020-06-30 at 15 50 42](https://user-images.githubusercontent.com/20446871/86134084-787fa900-bae9-11ea-93c8-33438a354ab2.png)